WhizzML.sublime-completions

```diff
@@ -178,6 +180,7 @@
         {"trigger": "matches?", "contents": "matches?"},
         {"trigger": "max\tCompute the maximum of", "contents": "max $1"},
         {"trigger": "max-key", "contents": "max-key"},
+        {"trigger": "md5\tGenerate the Hash (MD5) of a string", "contents": "md5 $1"},
```

Comment:
i would say "Compute the MD5 hash of a string"  (and same below with SHA1, SHA256)